manifests/a/Autodesk,Inc/AutodeskDownloadManager/4.0.19.0/Autodesk,Inc.AutodeskDownloadManager.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Autodesk,Inc.AutodeskDownloadManager

```

Comment:
The publisher already exists as `Autodesk` so the package ID here should be updated to match